test/presenters/base.presenter.test.js

```diff
@@ -186,6 +186,26 @@ describe('Base presenter', () => {
     })
   })
 
+  describe('#formatNumberAsMoneyWithCommas()', () => {
+    const valueInPence = 1149.5
```

Comment:
Should this be valueInPounds not pence? If it was pence it would be equal to 114950?

Reply:
Doh!